driver/src/clawfc/CLAWAddIgnoreScanner.g4

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is released under terms of BSD license
+ * See LICENSE file for more information
+ * @author Mikhail Zhigun
+*/
+/**
+ * ANTLR 4 Grammar file for recognizing CLAW Ignore directives.
+ */
+grammar CLAWAddIgnoreScanner;
```

Comment:
Why this is not part of the standard CLAW grammar.

Reply:
Ignore directive is processed at the preprocessing stage, well before actual CLAW processing.